source/platform/jmx.cpp

```diff
@@ -0,0 +1,653 @@
+#include "jmx_actions.h"
+#include "jmx.h"
+#include "mbed.h"
+#include "MicroBitHeapAllocator.h"
+#include "MicroBitCompat.h"
+
+JMXInitPacket* jmx_init_p = NULL;
+FSRequestPacket* jmx_fsr_p = NULL;
+DIRRequestPacket* jmx_dir_p = NULL;
+StatusPacket* jmx_status_p = NULL;
+RedirectPacket* jmx_redirect_p = NULL;
+UARTConfigPacket* jmx_uart_p = NULL;
```

Comment:
can these be made static?

p.s. what's the rationale for this bit being purely C rather than cpp? Nothing wrong with that, just less consistent with the restof the code.